pkg/bpf/bpf_restart.go

```diff
@@ -96,6 +96,8 @@ func SetStartStatus(versionMap *ebpf.Map) {
 		SetStartType(Restart)
 	} else {
 		log.Infof("kmesh start with Update")
+		// TODO : update mode has not been fully developed and is currently consistent with normal mode
+		log.Infof("Update mode support is under development, Will be started in Normal mode.")
```

Comment:
It's more appropriate to adjust the log level to warn

Reply:
I would prefer hidding this